### PR TITLE
Clean up TeacherHomepageTest - announcement, etc.

### PR DESCRIPTION
**Pull request by Erin007**
As a follow up to recent work refactoring the TeacherHomepageTest, this PR makes the assertions for the `HeaderBanner`, `ProtectedStatefulDiv` and `Announcements` more consistent with the shallow render tests for other components. 

Testing `Sections` in isolation is the next step in continuing to clean up this test.

### CODE REVIEW

File: apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js [hunk 1]
```diff
@@ -66,14 +58,15 @@ describe('TeacherHomepage', () => {
...
     );
-    const headerBanner = wrapper.childAt(0);
-    assert.equal(headerBanner.name(),'HeaderBanner');
-    assert.equal(headerBanner.props().headingText, 'Home');
-    assert.equal(headerBanner.props().extended, false);
+    const headerBanner = wrapper.find('HeaderBanner');
+    assert.deepEqual(headerBanner.props(), {
+      headingText: "Home",
+      short: true
+    });
```
Comment (islemaster) [inline apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js:65]: I've been wondering if it's nicer to express these as 
```jsx
expect(wrapper).to.containMatchingElement(
  <HeaderBanner
    headingText="Home"
    short
  />
);
```

I think it's a good assertion either way, I'll leave it up to you.

**Comment by islemaster** [comment]
Do we have unit tests for `HeaderBanner` and `Announcements` yet?  This would be a good time to add them if they don't already exist, since we're no longer fully rendering them in `TeacherDashboardTest`.

File: apps/test/unit/templates/NotificationTest.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+};
+
+const success = {
+  notice: "Wonder Woman Saved the Day",
+  details: "Things were pretty sketchy there for awhile, but don't worry- she's on top of it.",
+  dismissible: true
+};
+
+const failure = {
+  notice: "Lex Luther Attacked Metropolis",
```
Comment (islemaster) [inline apps/test/unit/templates/NotificationTest.js]: [`s/Luther/Luthor`](https://en.wikipedia.org/wiki/Lex_Luthor)

**Comment by ashercodeorg** [comment]
Unfortunately, Lex is not appeased, as his title (classification?) was not spelled correctly.  :)

(no action required - villain)

**Pull request merged**